nodejs/dist/uws.js

```diff
@@ -94,14 +94,27 @@ class Socket {
      *
      * @param {String|Buffer} message The message to send
      * @param {Object} options Send options
+     * @param {Function} cb optional callback
      * @public
      */
-    send(message, options) {
+    send(message, options, cb) {
         /* ignore sends on closed sockets */
-        if (!this.nativeSocket) return;
+        if(typeof options==='function'){
+            cb = options;
+            options = {};
+        }
+        if (!this.nativeSocket){ 
+            if(cb){
+                cb('Socket closed');
+            }
+            return;
+        }
 
         const binary = options && options.binary || false;
         this.server.nativeServer.send(this.nativeSocket, message, binary);
+        if(cb){
+            cb(null);
```

Comment:
It should be fine as the docs doesn't specify, also, it's kind of bullshit callback anyways because there is no way to know when things has been sent (especially in libuv where they queue the data in their own buffers and call that a "send").